tests/gentropy/conftest.py

```diff
@@ -459,22 +459,20 @@ def sample_finngen_studies(spark: SparkSession) -> DataFrame:
 
 
 @pytest.fixture()
-def sample_eqtl_catalogue_finemapping_credible_sets(spark: SparkSession) -> DataFrame:
+def sample_eqtl_catalogue_finemapping_credible_sets(session: Session) -> DataFrame:
     """Sample raw eQTL Catalogue credible sets outputted by SuSIE."""
-    return spark.read.option("delimiter", "\t").csv(
-        "tests/gentropy/data_samples/QTD000584.credible_sets.tsv",
-        header=True,
-        schema=EqtlCatalogueFinemapping.raw_credible_set_schema,
+    return EqtlCatalogueFinemapping.read_credible_set_from_source(
+        session,
+        credible_set_path=["tests/gentropy/data_samples/QTD000584.credible_sets.tsv"],
     )
 
 
 @pytest.fixture()
-def sample_eqtl_catalogue_finemapping_lbf(spark: SparkSession) -> DataFrame:
+def sample_eqtl_catalogue_finemapping_lbf(session: Session) -> DataFrame:
     """Sample raw eQTL Catalogue table with logBayesFactors outputted by SuSIE."""
-    return spark.read.option("delimiter", "\t").csv(
-        "tests/gentropy/data_samples/QTD000584.lbf_variable.txt",
-        header=True,
-        schema=EqtlCatalogueFinemapping.raw_lbf_schema,
+    return EqtlCatalogueFinemapping.read_lbf_from_source(
```

Comment:
Because the file name needed to be captured + some simple steps were performed by the `read` functions, I had to use the dedicated read functions for the tests. Hence, I had to pass the gentropy session object instead the spark session.